pkgs/games/enigma/default.nix

```diff
@@ -1,20 +1,20 @@
-{ stdenv, fetchurl, makeWrapper, pkg-config, curl, imagemagick, libpng, SDL, SDL_image, SDL_mixer, SDL_ttf, xercesc, xdg_utils, hicolor-icon-theme }:
+{ stdenv, fetchurl, makeWrapper, pkg-config, gettext, imagemagick, curl, libpng, SDL2, SDL2_image, SDL2_mixer, SDL2_ttf, xercesc, xdg_utils, hicolor-icon-theme }:
 stdenv.mkDerivation rec {
   pname = "enigma";
-  version = "1.21";
+  version = "1.30-alpha";
 
   src = fetchurl {
-    url = "mirror://sourceforge/enigma-game/enigma-${version}.tar.gz";
-    sha256 = "d872cf067d8eb560d3bb1cb17245814bc56ac3953ae1f12e2229c8eb6f82ce01";
+    url = "https://github.com/Enigma-Game/Enigma/releases/download/${version}/${pname}-${version}.tar.gz";
```

Comment:
We should probably use fetchFromGitHub.

Reply:
Note that we're fetching a release tarball, not a repository snapshot. (The repository snapshot is lacking some docfiles, if I remember correctly.) Does fetchFromGitHub work with release tarballs?

Reply:
I thought we fetched source.